lib/execution_engine2/SDKMethodRunner.py

```diff
@@ -862,7 +862,7 @@ def _job_state_from_jobs(jobs):
             mongo_rec = job.to_mongo().to_dict()
             mongo_rec["_id"] = str(job.id)
             mongo_rec["job_id"] = str(job.id)
-            mongo_rec["created"] = str(job.id.generation_time)
+            mongo_rec["created"] = str(job.id.generation_time.timestamp())
             mongo_rec["updated"] = str(job.updated)
```

Comment:
Do we want to return float string or just float here? I would prefer to return float value. Unless there is a requirement from Eric.